tests/environment/docker-compose-legacy-transforms-only.yml

```diff
@@ -0,0 +1,113 @@
+version: "3"
+
+# The acs-deployment project contains the product version of docker-compose.
+#
+# This version is for testing and development use. For testing and debug we need to have a more open version.
+# For remote debug we need the CATALINA_OPTS and to expose port 8000.
+
+# Add the following environment variable to CATALINA_OPTS to activate YourKit profiling agent on tomcat
+#-agentpath:/usr/local/YourKit-JavaProfiler-2018.04/bin/linux-x86-64/libyjpagent.so=delay=200000,listen=all,sessionname=$$HOSTNAME,dir=/tmp/Alfresco/yourkit,onexit=snapshot,periodicperf=600,periodicmem=600,probe_off=*
+
+services:
+    alfresco:
+        image: alfresco/alfresco-content-repository:latest
+        environment:
+            CATALINA_OPTS : "
+                -agentlib:jdwp=transport=dt_socket,address=*:8000,server=y,suspend=n
+                "
+            JAVA_OPTS : "
+                -Dalfresco.port=8082
+                -Daos.baseUrlOverwrite=http://localhost:8082/alfresco/aos
+                -Ddb.driver=org.postgresql.Driver
+                -Ddb.username=alfresco
+                -Ddb.password=alfresco
+                -Ddb.url=jdbc:postgresql://postgres:5432/alfresco
+                -Dalfresco.restApi.basicAuthScheme=true
+                -Dmessaging.broker.url=\"failover:(tcp://activemq:61616)?timeout=3000&jms.useCompression=true\"
+                
+                -Dtransform.service.enabled=false
+
+                -Dlocal.transform.service.enabled=false
+
+                -Dlegacy.transform.service.enabled=true
```

Comment:
Does it mean that you don't need the containers for the transformers as those are not going to be used?

Reply:
Tika and transform-misc containers were not required and have been removed.

Reply:
Sorry, I'm a bit confused now :)
So,
* if we enable local transformers, this means that the executables in content-repository docker container are used to do transformations. In this case the URLs and docker containers for individual transform engines are not required.
* transform service does not exist in ACS Community, there is no such property like `transform.service.enabled` and tests for it should not be required.
* if you remove some of the transform engines (misc and tika) and tests still work, it means that the tests are incomplete or there is some configuration issue. I don't think that compose should have some engines missing, it's either all of them or none at all.

Reply:
The `transform.service.enabled` has been removed as this is not a community feature

Reply:
These are integration tests, intended to confirm that the legacy/ local/ or the Transform service, depending on which properties are turned on, can communicate. 
The Legacy and Local transform services are able to communicate with T-engines and these tests are to make sure this communication is working correctly, so these containers are needed for the tests. 
As you rightly said, the transform service is only for enterprise and has not been added.